text/0037-syft-bom.md

```diff
@@ -0,0 +1,103 @@
+# Support for CycloneDX and Syft SBoMs in Paketo
+
+## Summary
+
+Buildpacks which directly provide a dependency or buildpacks which install
+application dependencies will support the generation of SBoM in the format of
+CycloneDX and Syft. Per [CNB
+RFC#95](https://github.com/buildpacks/rfcs/blob/main/text/0095-sbom.md), the
+SBoM documents will live at `<layer>.bom.<ext>.json`, `launch.bom.<ext>.json`
+and `build.bom.<ext>.json` where `<ext>` will be `cdx.json` (CycloneDX) or
+`syft.json` (Syft).
```

Comment:
```suggestion
and `build.bom.<ext>.json` where `<ext>` will be `cdx` (CycloneDX) or
`syft` (Syft).
```

Minor change. The `<ext>` part shouldn't have `.json` which is already part format listed.